main.cpp

```diff
@@ -79,42 +80,91 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
-
+bool playSoccer(bool soccerField, int soccerBall = 1 , int soccerPlayers = 2 )
+    {
+         ignoreUnused( soccerField , soccerBall , soccerPlayers ); 
+         return {};
+    }
 /*
  2)
- */
+ */
+ void petName(char firstLetterFirstName, char firstLetterLastName = 'H')
+ {
+     ignoreUnused( firstLetterFirstName , firstLetterLastName );
+ }

```

Comment:
`petName` is not an action, it's a noun.
Name functions what they do, name things what they are.
- [ ] revise